views/page-not-found.html

```diff
@@ -12,8 +12,11 @@ <h1 class="govuk-heading-l">Page not found</h1>
 
       <p class="govuk-body">If you pasted the web address, check you copied the entire address.</p>
 
-      <p class="govuk-body">If the web address is correct or you selected a link, the page may have moved. Go to the <a href="/register-an-overseas-entity">Register an overseas entity start page.</a></p>
+      <p class="govuk-body">If the web address is correct or you selected a link, the page may have moved. </p>
+      
+      <p class="govuk-body"> To register an overseas entity, go to the <a href="/register-an-overseas-entity">Register an overseas entity start page.</a> </p>
 
+      <p class="govuk-body"> To file an overseas entity update statement, go to the <a href="www.gov.uk/guidance/file-an-overseas-entity-update-statement">File an overseas entity update statement start page.</a> </p>
```

Comment:
If we followed what registration is doing then the last link would be: "/update-an-overseas-entity".
I've added a question to https://companieshouse.atlassian.net/browse/UAR-268